northstar/src/runtime/mount.rs

```diff
@@ -99,23 +102,11 @@ impl MountControl {
 
         task::spawn(async move {
             let start = time::Instant::now();
-
             let manifest = npk.manifest();
-            debug!("Mounting {}:{}", manifest.name, manifest.version);
-            let use_verity = key.is_some();
-
-            if npk.version() != &Manifest::VERSION {
```

Comment:
do we not need the version check anymore here?

Reply:
No because it's done when the npk is added to repository.